mso/resource_mso_schema_site_bd_subnet.go

```diff
@@ -105,7 +105,7 @@ func resourceMSOSchemaSiteBdSubnetImport(d *schema.ResourceData, m interface{})
 	found := false
 	stateBd := get_attribute[4]
 	stateIp := import_split[2]
-	for i := 0; i < count; i++ {
+	for i := 0; i < count && !found; i++ {
```

Comment:
@mike-guy Can we forego the **!found** in the loop condition by adding the following?

`terraform import mso_schema_site_bd_subnet.sub1 {schema_id}/site/{site_id}-{template_name}/bd/{bd_name}/ip/{ip}`
```
templateInfo := regexp.MustCompile("-").Split(get_attribute[2], 2)
site_template := strings.Split(get_attribute[2], "-")
stateSite := site_template[0]
stateTemplate := templateInfo[1]
apiTemplate := models.StripQuotes(tempCont.S("templateName").String())
if apiSite == stateSite && apiTemplate == stateTemplate {
```